modules/sbo-deploying-the-spring-petclinic-sample-application.adoc

```diff
@@ -10,81 +10,81 @@ To deploy the Spring PetClinic sample application on an {product-title} cluster,
 . Deploy the `spring-petclinic-rest` application with the `PostgresCluster` custom resource (CR) by running the following command in shell:
 +
 ----
-$ oc apply -f - << EOD
+$ oc apply -n my-petclinic -f - << EOD
 ---
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: spring-petclinic-rest
-  namespace: my-postgresql
+  name: spring-petclinic
   labels:
-    app: spring-petclinic-rest
+    app: spring-petclinic
 spec:
   replicas: 1
   selector:
     matchLabels:
-      app: spring-petclinic-rest
+      app: spring-petclinic
   template:
     metadata:
       labels:
-        app: spring-petclinic-rest
+        app: spring-petclinic
     spec:
       containers:
-        - name: application
-          image: quay.io/baijum/spring-petclinic-rest:latest
+        - name: app
+          image: quay.io/service-binding/spring-petclinic:latest
+          imagePullPolicy: Always
           env:
           - name: SPRING_PROFILES_ACTIVE
-            value: postgresql,spring-data-jpa
+            value: postgres
           ports:
           - name: http
-            containerPort: 9966
+            containerPort: 8080
 ---
 apiVersion: v1
 kind: Service
 metadata:
-  name: spring-petclinic-rest
-  namespace: my-postgresql
+  labels:
+    app: spring-petclinic
+  name: spring-petclinic
 spec:
+  type: NodePort
   ports:
-  - port: 80
-    targetPort: 9966
+    - port: 80
+      protocol: TCP
+      targetPort: 8080
   selector:
-    app: spring-petclinic-rest
+    app: spring-petclinic
 EOD
 ----
 +
 The output verifies that the Spring PetClinic sample application is created and deployed:
 +
 .Example output
 ----
-deployment.apps/spring-petclinic-rest created
-service/spring-petclinic-rest created
-----
-
-. Set up port forwarding from the application port to access the sample application from your local environment:
-+
-----
-$ oc port-forward --address 0.0.0.0 svc/spring-petclinic-rest 9966:80 -n my-postgresql
+deployment.apps/spring-petclinic created
+service/spring-petclinic created
 ----
-
-. Access link:http://localhost:9966/petclinic[http://localhost:9966/petclinic].
-+
-You can now remotely access the Spring PetClinic sample application at localhost:9966.
 +
 [NOTE]
 ====
-The application is not yet connected to the database service. If you try to interact with the application, it will return errors.
+If you are deploying the application using *Container images* in the *Developer* perspective of the web console, you must enter the following environment variables under the *Deployment* section of the *Advanced options*:
+
+
+* Name: SPRING_PROFILES_ACTIVE
+* Value: postgres
 ====
+
+. Verify that the application is not yet connected to the database service by running the following command (it will take a few minutes):
```

Comment:
I think you might have worded "it will take a few minutes" better in a previous example.
Our style guidelines ask us to avoid using parenthesis like this. The wording is unclear about whether running the command takes a few minutes, or the application takes a few minutes to connect to the database.